packages/@webex/internal-plugin-metrics/src/rtcMetrics/index.ts

```diff
@@ -160,6 +168,21 @@ export default class RtcMetrics {
    * @returns {void}
    */
   private sendMetrics() {
+    const metricsAttributes = {
+      type: 'webrtc',
+      version: '1.1.0',
+      userId: this.webex.internal.device.userId,
+      correlationId: this.correlationId,
+      connectionId: this.connectionId,
+      data: this.metricsQueue,
+    };
+
+    if (this.meetingId) {
+      metricsAttributes['meetingId'] = this.meetingId;
+    } else if (this.callId) {
+      metricsAttributes['callId'] = this.callId;
```

Comment:
Adding these conditionally as we don't want to sent the key `meetingId` or `callingId` with value as `undefined`. A key will only be added if the value exists for it.